packages/flutter/lib/src/material/divider.dart

```diff
@@ -12,11 +13,11 @@ import 'theme.dart';
 /// In the material design language, this represents a divider.
 ///
 /// Dividers can be used in lists, [Drawer]s, and elsewhere to separate content
-/// vertically. To create a one-pixel divider between items in a list, consider
-/// using [ListTile.divideTiles], which is optimized for this case.
+/// vertically or horizontally. To create a one-pixel divider between items in
```

Comment:
Would help to include something here about this being based on the [axis] property